packages/terra-tag/src/Tag.jsx

```diff
@@ -66,6 +72,18 @@ class Tag extends React.Component {
     }
   }
 
+  handleClick(event) {
+    if (this.props.onTagClick) {
+      this.props.onTagClick(event.target.id);
+    }
+    if (this.props.onClick) {
+      this.props.onClick();
+    }
+    setTimeout(() => {
+      this.setState({ focused: true });
+    }, 100);
```

Comment:
Why do we need a setTimeout?

Reply:
I guess this was used for focus issue. Have removed this. Thanks